src/Orbit/main.cpp

```diff
@@ -193,9 +193,9 @@ static std::optional<std::string> GetCollectorPath(const QProcessEnvironment& pr
   return std::nullopt;
 }
 
-static orbit_qt::DeploymentConfiguration FigureOutDeploymentConfiguration() {
+static DeploymentConfiguration FigureOutDeploymentConfiguration() {
```

Comment:
Would it make sense to move this function into the new session setup module?

Reply:
yes, done!